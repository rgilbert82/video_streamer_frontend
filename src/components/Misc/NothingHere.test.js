import React           from 'react';
import { shallow }     from 'enzyme';
import { NothingHere } from '.';

describe(NothingHere, () => {
  let component;

  beforeEach(() => {
    component = shallow(<NothingHere />);
  });

  it('renders properly', () => {
    expect(component).toMatchSnapshot();
  });
});
