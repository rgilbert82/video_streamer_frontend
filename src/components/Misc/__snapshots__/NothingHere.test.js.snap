// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class NothingHere extends _react2.default.Component {
  render() {
    return (
      _react2.default.createElement('div', { __source: { fileName: _jsxFileName, lineNumber: 6 }, __self: this },
        _react2.default.createElement('p', { __source: { fileName: _jsxFileName, lineNumber: 7 }, __self: this }, 'Oops, nothing here!')));


  }} renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NothingHere />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        Oops, nothing here!
</p>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Oops, nothing here!",
      },
      "ref": null,
      "rendered": "Oops, nothing here!",
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          Oops, nothing here!
</p>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Oops, nothing here!",
        },
        "ref": null,
        "rendered": "Oops, nothing here!",
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
